content/en/authors/aatmaj-janardanan/_index.md

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
To have an authors page that displays blog posts, `bio` will be replaced with `authors`. We also have to follow this file structure (`content/en/authors/author-name/_index.md`) when adding a new author/bio to the site. I'll create a separate story to remove the `bio` files.

Reply:
Could we also do a redirect that makes sure if someone goes to one of the old bio pages, (eg `https://rotational.io/bio/rebecca-bilbro/`) they get rerouted to whatever the new url will be so they don't see a broken link?

Reply:
Sure, I'll include that as part of the acceptance criteria when deleting the bio files.